### ISSUE TITLE
[9.5.0-rc1] Invalid LDAP authentication sources cause the login process to fail with a white page.

### ISSUE THREAD

**Issue by eiseli** [glpi-project/glpi#7431]
**Describe the bug**
Invalid LDAP authentication sources cause the login process to fail with a white page.

**Page(s) URL**

front/login.php

**To reproduce**

A. Prerequesites:
1. have a ldap source that doesn't work (for us it was an inexisting OU in active directory)
2. in Setup > General > General setup set the value for "Display source dropdown on login page" to "No"
3. Have a user whose authentication source is a valid ldap.
B. Steps:
1. Log out of GLPI
2. In the login screen, input username and password of the user described above in A3.
3. Submit the form by clicking the button "Post"

**Expected behavior**
1. The login process should look up the user's login source and successfully log him in
2. In the eventuality the user is not found in the saved login source (and only then), cycle through all available LDAP sources
3. If there's an error with the ldap source, fail gracefully for the user (for example "Permanent authentication error, please get in touch with your support")
4. In the log file, there should be some more information, showing the basedn and the filter values.

**Logs**

In the php-errors.log one gets the following message:

Uncaught Exception RuntimeException: Something went wrong searching in LDAP directory in /srv/glpi/9.5.0-rc1/inc/authldap.class.php at line 2954

**Suggested fix for the expected behavior point 4**
```diff
--- authldap.class.php  2020-06-15 15:01:43.639167754 +0200
+++ authldap.class.bak.php      2020-06-10 13:49:57.000000000 +0200
@@ -2951,7 +2951,7 @@
          }
          return false;
       }
-      throw new \RuntimeException('Something went wrong searching in LDAP directory. (basedn='.$values['basedn'].' filter='.$filter.'ldap_parameters='.$ldap_parameters.')');
+      throw new \RuntimeException('Something went wrong searching in LDAP directory');
    }
```

**Comment by eiseli** [comment]
Note that this bug already exists in 9.4.5

**Comment by eiseli** [comment]
Oh, I just notice, I made the diff the wrong way around.

**Comment by orthagh** [comment]
no issue :p, i push a pr with your fix in the correct order :)

**Comment by fbsdev** [comment]
Note that this bug already exists in 9.5.2...

**Comment by cedric-anne** [comment]
> Note that this bug already exists in 9.5.2...
> 
> I using the filter
> 
> `(&(objectCategory=person)(objectClass=user)(!(userAccountControl:1.2.840.113556.1.4.803:=2))(mail=*))`
> 
> Rgs

Hi,

A fix has been made in e5b1ab0d82cb9b37d2f6c51550aa6584b606728f. You may encounter another bug. Please open a new issue, with all requequested informations.

Regards
